docs/DeveloperGuide.md

```diff
@@ -834,3 +834,6 @@ Common prerequisite: List all elderly using the `list` command.
    1. Delete the `data/contactmate.json` file.
    2. Launch the app.<br>
        Expected: The app should show a list of sample persons.
+
+3. **Mark elderly as called in the history view:** Currently, users can view the call history and profile view of the elderly in the history view together. This allows users to view the NRIC of the elderly, which may give the wrong impression that they can mark the elderly using NRIC as called in the history view (e.g. mark SXXXXXXXH). However, this is unintended as the history view is meant for viewing purposes only. We plan to provide the ability to mark the elderly whose details is specified in the history view as called.
```

Comment:
I think can focus less on why we implemented it like that initially, and focus more on the enhancement

Reply:
also I think a better enhancement can be to allow **modification** (i.e. allow edit also), but same as what you said ah, only for that specific elderly

Reply:
I tried to phrase it in a way where it sounds less like a bug? \

good idea on phrasing multiple commands.

Reply:
>I tried to phrase it in a way where it sounds less like a bug? \

ya its overall okay, maybe omit this line? `However, this is unintended as the history view is meant for viewing purposes only.`